include/translatable-object-with-types-trait.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Trait to use for objects that can have one or more types.
+ * This must be used with {@see PLL_Translatable_Object_With_Types_Interface}.
+ *
+ * @since 3.4
+ *
+ * @property string[] $db {
```

Comment:
Maybe it would be good to declare `$db` here too.
As I understand the `@property` tag is to use when using magic methods: https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/property-read.html#property-property-read-property-write

Reply:
I tried that first, but PHPStan doesn't understand that it must use the `$db` from the trait, and uses the one from the parent class instead, leading to an unknown key `$this->db['type_column']`.

Reply:
However, I added `@property` only to document `$type_column`, so we could remove it and keep only `@phpstan-property` (but I'd prefer keep both).